### DISCUSSION TITLE
Autoscaled Application on M1 MacBook Air, after v1.10.3

### DISCUSSION THREAD

**Discussion by mathogre** [freeplane/freeplane#1354] [Help] [answered]
If I install a version of Freeplane beyond v1.10.3, the application is autoscaled as if zoomed 2x or more.  Everything is zoomed, not just the mind map contents.

I've searched here and via Google for solutions but have found nothing.

I have tried v1.11.5 for Apple and for Intel, and in both cases the application is zoomed.  I've updated Java (JRE, not JDK) for both Apple and Intel, uninstalling as per Oracle instructions, and nothing helps.  At present I'm actually running without Java, so I suspect Java is already in the Freeplane installation dmgs?

I'm back to running 1.10.3, and it looks fine, but I'd really like to be running the latest.

Any insights would be greatly appreciated!  Thank you!

**Comment by dpolivaev** [comment]
Upload screenshots and zipped auto.properties.
Yes, Java us bundled. It is a requirement by Apple.

**Reply by mathogre** [reply]
Here are the screenshots, one with v1.10.3 and the other with v1.11.5.  Where do I find the auto.properties file you need?

<img width="1440" alt="v1 10 3" src="https://github.com/freeplane/freeplane/assets/68301018/2f87ee10-b161-4f5f-9323-12562b84620b">
<img width="1440" alt="v1 11 5" src="https://github.com/freeplane/freeplane/assets/68301018/8f1b4364-be49-467c-80a5-bfac9a1b30c9">

**Reply by dpolivaev** [reply]
Tools->Open user directory

**Reply by mathogre** [reply]
Got it.  Thanks!  Here it is.

[auto.properties.zip](https://github.com/freeplane/freeplane/files/12401137/auto.properties.zip)

**Reply by dpolivaev** [reply]
Can you please tell me what is your real monitor size?
It looks like it is about 13.3 inch, is it correct?

**Reply by mathogre** [reply]
Yes.  It's 13.3.  Res is 2560x1600.  Retina.

**Reply by dpolivaev** [reply, marked as answer]
It perfectly explains your observations.

In your auto.properties your monitor size was set to 22 inches and it lets all elements be 13.3/22 times smaller than they should be,
In the current version Freeplane on MacOS trusts the information it gets from the OS and ignores your old settings.

But if you want to let Freeplane behave as earlier you can manually edit auto.properties and add a line
`````
monitor_size_inches.hide=false
`````

Make sure Freeplane does not run when you edit and save auto.properties.

**Reply by mathogre** [reply]
YES!  It works perfectly now.  Thank you very much!!!!
